core/src/main/java/com/onelogin/saml2/authn/SamlResponse.java

```diff
@@ -241,6 +241,15 @@ public boolean isValid(String requestId) {
 								+ ", does not match the ID of the AuthNRequest sent by the SP: " + requestId, ValidationError.WRONG_INRESPONSETO);
 				}
 
+				// Check SAML version on the response
```

Comment:
// Check SAML version on the assertion

Reply:
Whoops, you're right, sorry! I'm fixing it.